### PR TITLE
Localize hi.toml

### PR DESCRIPTION
**Pull request by panchal20bhavin**
### Describe your changes
Translate Remaining English String into hindi language

### Related issue number or link (ex: `resolves #issue-number`)
#2182 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: i18n/hi.toml [hunk 1]
```diff
@@ -20,21 +20,21 @@ other = "में"
 
 # Phrases for tags
 [ui_see_all_tags]
-other = "See all tags"
+other = "देखना सभी टैग"
 [ui_tag]
-other = "Tag"
+other ="टैग"
```
Comment (jihoon-seo) [inline i18n/hi.toml:25]: We don't need to remove the spaces. Let's preserve.

```suggestion
other = "टैग"
```

File: i18n/hi.toml [hunk 2]
```diff
@@ -45,7 +45,7 @@ other = "गोपनीयता नीति"
 
 
 [footer_hub_button_text]
-other = "All CNCF Sites"
+other ="सभी CNCF साइटें"
```
Comment (jihoon-seo) [inline i18n/hi.toml:48]:
```suggestion
other = "सभी CNCF साइटें"
```

**Comment by jihoon-seo** [comment]
@Garima-Negi @sayantani11 @anubha-v-ardhan @jayesh-srivastava please review the Hindi localization in this PR!

File: i18n/hi.toml [hunk 3]
```diff
@@ -20,21 +20,21 @@ other = "में"
 
 # Phrases for tags
 [ui_see_all_tags]
-other = "See all tags"
+other = "देखना सभी टैग"
```
Comment (aj11anuj) [inline i18n/hi.toml:23]:
```suggestion
other = "सभी टैग को देख़े"
```

File: i18n/hi.toml [hunk 4]
```diff
@@ -20,21 +20,21 @@ other = "में"
...
 [ui_tag]
-other = "Tag"
+other = "टैग"
 [ui_tags]
-other = "Tags"
+other = "टैग"
```
Comment (aj11anuj) [inline i18n/hi.toml:27]:
```suggestion
other = "टैग्स"
```

File: i18n/hi.toml [hunk 5]
```diff
@@ -20,21 +20,21 @@ other = "में"
...
 [ui_search_by_tags]
-other = "Browse by Tags"
+other = "ब्राउज़ द्वारा टैग"
```
Comment (aj11anuj) [inline i18n/hi.toml:29]:
```suggestion
other = "टैग द्वारा ब्राउज करें"
```

File: i18n/hi.toml [hunk 6]
```diff
@@ -20,21 +20,21 @@ other = "में"
...
 [ui_tags_intro]
-other = "We've categorized the glossary terms. Use the filters to browse terms by tag."
+other = "हमने शब्दावली शब्दों को वर्गीकृत किया है. टैग द्वारा शब्दों को ब्राउज़ करने के लिए फ़िल्टर का उपयोग करें."
```
Comment (aj11anuj) [inline i18n/hi.toml:31]:
```suggestion
other = "हमने शब्दावली के शब्दों को वर्गीकृत किया है। टैग द्वारा शब्दों को ब्राउज़ करने के लिए फ़िल्टर का उपयोग करें।"
```

File: i18n/hi.toml [hunk 7]
```diff
@@ -20,21 +20,21 @@ other = "में"
...
 [ui_or_search_by_tags]
-other = "...or browse by tag"
+other = "...या ब्राउज़ द्वारा टैग"
```
Comment (aj11anuj) [inline i18n/hi.toml:33]:
```suggestion
other = "...या टैग द्वारा ब्राउज़ करें"
```

File: i18n/hi.toml [hunk 8]
```diff
@@ -20,21 +20,21 @@ other = "में"
...
 [ui_select_all]
-other = "Select All"
+other = "सबका चुनाव करना"
```
Comment (aj11anuj) [inline i18n/hi.toml:35]:
```suggestion
other = "सबका चुनाव करें"
```

File: i18n/hi.toml [hunk 9]
```diff
@@ -20,21 +20,21 @@ other = "में"
...
 [ui_deselect_all]
-other = "Deselect All"
+other = "सबका चुनाव रद्द करें"
```
Comment (aj11anuj) [inline i18n/hi.toml:37]:
```suggestion
other = "सारे चुनाव रद्द करें"
```

**Review by aj11anuj — CHANGES_REQUESTED** [review]
Some suggestions from my side

**Review by kumarankit999 — COMMENTED** [review]
Now looks good by anuj

**Review by aj11anuj — APPROVED** [review]
Lgtm 👍

**Review by seokho-son — APPROVED** [review]
LGTM. :)

FYI, to apply the latest approvers (the https://github.com/cncf/glossary/blob/main/CODEOWNERS#L43 ) to dev-hi, dev-hi needs to be rebased. 

After that, /i18n/hi.toml can be approved solely by Hindi team.

**Pull request merged**